invenio_records_resources/services/uow.py

```diff
@@ -183,6 +183,27 @@ def on_commit(self, uow):
             self._indexer.index(self._record, arguments=arguments)
 
 
+class RecordBulkCommitOp(Operation):
+    """Record bulk commit operation with indexing."""
+
+    def __init__(self, records, indexer=None, index_refresh=False):
+        """Initialize the bulk record commit operation."""
+        self._records = records
+        self._indexer = indexer
+        self._index_refresh = index_refresh
+
+    def on_register(self, uow):
+        """Save objects to the session."""
+        for record in self._records:
```

Comment:
do we really need this? not sure it is very useful, compared to each record.commit op

Reply:
Not sure I fully understand this comment, sorry ... but the point of RecordBulkCommitOp is to be able to bulk_index everything at once, after commiting it ot the DB, alternatively we could register the RecordCommitOp for each record (without the indexer) and register at the end the `RecordBulkIndexOp` to bulk index everything but I thought having on Op that does it together would be clearer